src/argilla/client/sdk/users/api.py

```diff
@@ -12,10 +12,155 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+from typing import List, Optional, Union
+
+import httpx
+
 from argilla.client.sdk.client import AuthenticatedClient
-from argilla.client.sdk.users.models import User
+from argilla.client.sdk.commons.errors_handler import handle_response_error
+from argilla.client.sdk.commons.models import (
+    ErrorMessage,
+    HTTPValidationError,
+    Response,
+)
+from argilla.client.sdk.users.models import UserCreateModel, UserModel
+from argilla.server.models import UserRole
+
+
+# TODO(alvarobartt,frascuchon): use `httpx.Client` instead of `AuthenticatedClient` and
+# fix mock in `tests/conftest.py` to use `httpx.Client` instead of `AuthenticatedClient`
+def whoami(
+    client: AuthenticatedClient,
+) -> UserModel:
+    # ) -> Response[Union[UserModel, ErrorMessage, HTTPValidationError]]:
+    """Sends a GET request to `/api/me` endpoint to get the current user information.
+
+    Args:
+        client: the authenticated Argilla client to be used to send the request to the API.
+
+    Returns:
+        A `UserModel` instance with the current user information.
+    # Returns:
+    #     A `Response` object containing a `parsed` attribute with the parsed response if
+    #     the request was successful, which is an instance of `UserModel`.
```

Comment:
I guess that we will use this once we update the function to return `Response` instead of `UserModel`. In the meantime, can we remove or move this comment to inside of the function? It will get rendered in the user docstring.